indico/modules/auth/__init__.py

```diff
@@ -30,6 +32,12 @@
 @multipass.identity_handler
 def process_identity(identity_info):
     logger.info('Received identity info: %s', identity_info)
+
+    if session_expiry := identity_info.multipass_data.get('session_expiry'):
```

Comment:
Hi @cbartz, please does this work when logging in with a local account? I'm getting a `AttributeError: 'NoneType' object has no attribute 'get'` when logging in with a local account.

Reply:
the `multipass_data` is `None` for local accounts. so this is currently a bug in the PR

Reply:
Okay, thanks